mne/channels/channels.py

```diff
@@ -207,16 +208,25 @@ def compensation_grade(self):
         """The current gradient compensation grade."""
         return get_current_comp(self.info)
 
-    def get_channel_types(self):
+    @fill_doc
+    def get_channel_types(self, picks=None, unique=False, only_data_chs=False):
         """Get a list of channel type for each channel.
 
+        Parameters
+        ----------
+        %(picks_nostr)s
```

Comment:
I don't think this should be `nostr`, that's usually reserved for when `info` is unavailable...?

Reply:
it was accurate given how the method was defined when you reviewed. Originally it only handled integer picks, and I figured YAGNI for string-based picking, since if you know the channel name or type you want to pick, you probably know its type already. But now I realize it was easy enough to add string-based picking, and also decided there's a real use case, e.g., disambiguating mag vs grad among `MEG_xxxx` channel names.  So now that works, and I've changed `picks_nostr` to `picks`.